src/NUnitFramework/tests/TestUtilities/Collections/SimpleEnumerable.cs

```diff
@@ -54,4 +54,59 @@ public IEnumerator GetEnumerator()
 
         #endregion
     }
+
+    class SimpleEnumerableWithIEquatable : IEnumerable<object>, IEquatable<SimpleEnumerableWithIEquatable>
+    {
+        public List<object> Contents { get; private set; } = new List<object>();
```

Comment:
Why the initializer?

Both constructors initialize the value of Contents, so this default initialization is redundant.

Reply:
Good catch; also the private set is unnecessary.

Reply:
Just saw this, should I write a PR to do this small refactor?  Don't mind, just not sure if it's "PR worthy" or something like that since it is pretty minor.

Reply:
If you're willing, a small PR would be great. 😄

Reply:
I agree. If something just makes sense to do, I don't care how small it is. =)